core/templates/suggestion-form.html

```diff
@@ -2,9 +2,10 @@
 
 {% block content %}
 {% if bra_size %}
+<h1>Your estimated bra size: <strong>{{bra_size}}</strong></h1>
 <p>{{bra_size}}</p>
```

Comment:
can remove line 6 because you say bra size in 5

Reply:
Clinton made a comment to Juan and I after the thing this morning that it would be important to have the bra size be really big on the suggestion page because that is what it is all about.

Reply:
I see, I took out line six instead.